libcaf_net/caf/net/http/client.test.cpp

```diff
@@ -129,7 +129,7 @@ struct fixture {
     auto client = net::http::client::make(std::move(app));
     auto transport = net::octet_stream::transport::make(fd2, std::move(client));
     auto mgr = net::socket_manager::make(mpx.get(), std::move(transport));
-    mpx->start(mgr);
+    std::ignore = mpx->start(mgr);
```

Comment:
Same for the changes in this and the other files: I think the tests should fail if `start` returns `false`.